doc/manual/cookbooks/phase_diagram/doc/phase_diagram.tex

```diff
@@ -22,7 +22,9 @@ \subsubsection{Phase diagram}
 
 \par The material model is then set up to mimic mantle phase transitions at 410, 520, and 660 km.% setup of mantle transitions
 Details of phase transitions are taken from \cite{billen2018decoupling}.
-A trick of assigning the values of density to the field of thermal capacity is used here [MIB: How is this done? can you be more explicit]. % trick
+A trick of assigning the values of density to the field of thermal capacity is used here. % [MIB: How is this done? can you be more explicit, addressed]. % trick
+To do this, I used the phase inputs implemented in the ViscoPlastic file.
+There, one can either assign values to individual phases with the separator '|', or assign constant values to all phases by a single number.
```

Comment:
I don't think this line (27) is actually needed. I was confused about where the trick in line 25 was happening.